### ISSUE TITLE
Do you report the top5 error for the encoder network?

### ISSUE THREAD

**Issue by oliver-im** [Eromera/erfnet_pytorch#20]
Hi,

While I was reading your paper, the paper mentioned that the encoder was trained using two strategies: "from scratch", and "pretrained".

I was wondering what was the top 5 error for the encoder when training on ImageNet. Is it comparable (or better) than other efficient architectures like mobilenet or xception?

**Comment by ChengshuLi** [comment]
@Eromera I am also interested in learning your top 5 error for ImageNet training. I am tweaking your encoder architecture to try to improve the performance. Therefore, I need to train different experiments on ImageNet and it would be very helpful to have your number as a reference. Thanks a ton!

**Comment by Eromera** [comment]
Hi! Sorry for the late reply. I don't have the top-5 error but I remember that top-1 accuracy was around 65%. This is fairly low but reasonable because the classifier that I add up is small (only one fully connected layer) and I don't increase the number of features or layers because the only purpose is to pretrain the weights and not to obtain a competitive accuracy in imagenet, so with a few modifications this result should go much higher.
